doc/host_config_schema/cchost_config.json

```diff
@@ -0,0 +1,412 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "title": "CCF node configuration",
+  "description": "JSON schema for configuration of a single CCF node (cchost executable)",
+  "properties": {
+    "enclave": {
+      "type": "object",
+      "properties": {
+        "file": {
+          "type": "string",
+          "description": "Path to enclave application"
+        },
+        "type": {
+          "type": "string",
+          "enum": ["release", "debug", "virtual"],
+          "default": "release",
+          "description": "Type of enclave application"
+        }
+      },
+      "description": "This section includes configuration for the enclave application launched by this node",
+      "required": ["file"]
+    },
+    "network": {
+      "type": "object",
+      "properties": {
+        "node_to_node_interface": {
+          "type": "object",
+          "properties": {
+            "bind_address": {
+              "type": "string",
+              "description": "Local address the node binds to and listens on"
+            }
+          },
+          "description": "Address (host:port) to listen on for incoming node-to-node connections (e.g. internal consensus messages)",
+          "required": ["bind_address"]
+        },
+        "rpc_interfaces": {
+          "type": "object",
+          "additionalProperties": {
```

Comment:
I haven't seen this style before. Why not move the inner bits out?

Reply:
I followed this approach: https://stackoverflow.com/a/27375654 This is because there can be any number of RPC interfaces (none might be OK too?).

Reply:
Gotcha, misunderstanding.